test/PowerShellEditorServices.Test/Debugging/DebugServiceTests.cs

```diff
@@ -815,6 +815,34 @@ public async Task DebuggerVariableHashtableDisplaysCorrectly()
             this.powerShellContext.AbortExecution();
         }
 
+        [Fact]
+        public async Task DebufferVariableNullStringDisplaysCorrectly()
```

Comment:
These tests don't actually get run in CI as of right now because of the large migration but you don't need to worry about that. I'm going to enable them again soon.

Thanks for adding this!

Reply:
I tried to run it locally but honestly wasn't sure if it actually ran, had no failures (that were related to this though).